spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/SparkTemplate.scala

```diff
@@ -4,7 +4,16 @@ import org.apache.spark.{SparkEnv, SparkConf, SparkContext}
 
 trait SparkTemplate {
   val conf = SparkTemplate.conf
-  val sc = SparkTemplate.sc
+  def sc = SparkTemplate.sc
+
+  def resetSparkContext(conf: SparkConf = conf): Unit = {
```

Comment:
If there is at most one SparkContext, then this method really belongs to SparkTemplate *object*.